WalletWasabi.Fluent/ViewModels/Wallets/WalletSettingsViewModel.cs

```diff
@@ -41,11 +43,22 @@ private WalletSettingsViewModel(IWalletModel wallet)
 				wallet.Settings.PreferPsbtWorkflow = value;
 				wallet.Settings.Save();
 			});
+
+		this.WhenAnyValue(x => x._wallet.Name).BindTo(this, x => x.WalletName);
+		
+		RenameCommand = ReactiveCommand.Create(OnRenameWallet);
 	}
 
+	public ICommand RenameCommand { get; set; }
+
 	public bool IsHardwareWallet { get; }
 
 	public bool IsWatchOnly { get; }
 
 	public ICommand VerifyRecoveryWordsCommand { get; }
+	
+	private void OnRenameWallet()
+	{
+		Navigate().To().WalletRename(_wallet, NavigationTarget.CompactDialogScreen);
```

Comment:
Set this value in the `NavigationTarget` property of the `[NavigationMetadata]` attribute of the `WalletRenameViewModel` class, so that callers can just do

`Navigate().To().WalletRename(_wallet);`

without having to specify navigation target explicitly

Reply:
Cool! Done :)